Command/SyncProjectCommand.php

```diff
@@ -236,29 +231,23 @@ private function hasEnvFileSupport($env = 'local')
         return file_exists(sprintf('.env.%s.dist', $env));
     }
 
-    private function getDatabaseConfig($environment, $hasEnvFileSupport)
+    private function getDatabaseConfig(string $environment, bool $hasEnvFileSupport)
     {
-        if (!$hasEnvFileSupport) {
-            $file = 'local' === $environment ?
-                sprintf('%s/config/parameters.yml', $this->rootDir) :
-                sprintf('%s/config/parameters.%s.yml.dist', $this->rootDir, $environment)
-            ;
-
-            if (!file_exists($file)) {
-                throw new LogicException(sprintf('No parameters file for environment "%s" found.', $environment));
-            }
+        $file = sprintf('%s/../../.env.local', $this->rootDir);
```

Comment:
Is this really true? It seems to me, that `rootDir` is either `projectDir` or `projectDir/src`. Either way `../..` would result in a wrong directory, no?

Reply:
rootDir was `~/dev/naturmetropole-graubuenden/src/HttpKernel/` so this version worked, however `projectDir` is the better choice anyway.